e2e/rollup/src/rollup.test.ts

```diff
@@ -172,7 +174,7 @@ describe('Rollup Plugin', () => {
     );
     updateFile(`libs/test/package.json`, JSON.stringify({ name: 'test' }));
     updateFile(
-      `libs/test/rollup.config.js`,
+      `libs/test/rollup.config.mjs`,
```

Comment:
Rollup v2 must have been doing something to process the file, but with v4 it is using Node resolution so the file must be `.mjs` or the workspace needs `type: 'module'` in `package.json`.